prac_07/guitar.py

```diff
@@ -0,0 +1,17 @@
+class Guitar:
+    """Guitar information"""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name}, ({self.year}) : ${self.cost}"
+
+    def get_age(self, current_year=2022):
+        return current_year - self.year
```

Comment:
Nice use of a default value. You could also use a constant here, but I like what you've done